src/main/java/pong/PaddleKeyListener.java

```diff
@@ -15,8 +15,8 @@ public PaddleKeyListener(Paddle paddle) {
     public void keyPressed(KeyEvent e) {
         super.keyPressed(e);
         switch (e.getKeyCode()) {
-            case KeyEvent.VK_DOWN -> paddle.move(Direction.South);
-            case KeyEvent.VK_UP -> paddle.move(Direction.North);
+           //case KeyEvent.VK_DOWN -> paddle.move(Direction.South);
+            //case KeyEvent.VK_UP -> paddle.move(Direction.North);
```

Comment:
We still need some sort of key listener. The paddle class is going to need a method to move and the key listener is going to need to tell it to move up or down. Since we're no longer using the Directions class, maybe something like paddle.move(1) for up and paddle.move(-1) for down?

Reply:
Yes! That is a great idea. Thank you. I will incorporate this when I work on the Paddle. I can include it in this pr or I can add it to the next

Reply:
Including it in the paddle pr probs makes more sense. Thank you!